notebooks/15_polygon_fill.ipynb

```diff
@@ -67,6 +67,7 @@
     "\n",
```

Comment:
You can have multiple geometries in one geodataframe and it will just handle it for you 😄 The looping is already per polygon inside the `fast_polygon_fill` function. 

Lmk if we need to change anything in the markdown or docs to make it clearer!

Reply:
Ahh i think ill have to edit the function docstring